### DISCUSSION TITLE
Recommended HW (DAC) for RPi3B+

### DISCUSSION THREAD

**Discussion by shumafuk** [dtcooper/raspotify#501] [Q&A] [answered]
Please, can you recommend me DAC for RPi3B+ that will be used generally for Raspotify? It will be connected to KALI AUDIO LP-6 V2 pair of monitors.

Unfortunately I'm not able to get Raspotify working in docker on my NAS, that has better audio output.

**Comment by JasonLG1979** [comment, marked as answer]
Pretty much any USB DAC will work. Topping makes some really good USB DACs so I've heard. Hifiberry also has a few decent DAC hats.
